File: Java/OLD/context-tasks/src/main/java/org/cqrs101/tasks/commands/CreateTask.java
package org.cqrs101.tasks.commands;

import java.util.Date;
import java.util.UUID;
import org.cqrs.Command;

public class CreateTask extends Command {

    private UUID id;
    private String title;
    private String description;
    private int priority;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getPriority() {
        return priority;
    }

    public void setPriority(int priority) {
        this.priority = priority;
    }
}

File: Net/Step0/Tasks.Domain/Commands/CreateTask.cs
using Cqrs;
using System;

namespace Tasks.Commands
{
    public class CreateTask : ICommand
    {
        public DateTime? DueDate { get; set; }
        public Guid Id { get; set; }
        public String Title { get; set; }
        public String Description { get; set; }
        public int Priority { get; set; }
    }
}
